bundles/org.eclipse.ltk.ui.refactoring/plugin.properties

```diff
@@ -29,3 +29,7 @@ renameResource.description=Rename the selected resource and notify LTK participa
 renameResource.commandParameter.newName=Selected resource's new name.
 
 contentMergeViewers.refactoring.label=Refactoring Compare
+
+copyProject.name = Copy Project
+copyProject.commandParameter.newName = Selected project's new name.
+copyProject.commandParameter.newLocation = Location of the new project.
```

Comment:
Would it be better to try to have both sentence follow the same pattern, eg "The (name|location) for the new project" ?

Reply:
yes, i adapted it